php/WP_CLI/Runner.php

```diff
@@ -247,6 +247,8 @@ private function cmd_starts_with( $prefix ) {
 	public function find_command_to_run( $args ) {
 		$command = \WP_CLI::get_root_command();
 
+		WP_CLI::do_hook( 'find_command_to_run_pre' );
```

Comment:
Please add a reference to this hook in the doc block of `WP_CLI::add_hook()`: https://github.com/wp-cli/wp-cli/blob/3ccde5b05bb42c4ce4213e636e96d36e96f24bc3/php/class-wp-cli.php#L206-L215